README.rst

```diff
@@ -16,7 +16,7 @@ For the latest version:
 
 ::
 
-  addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.1.0")
+  addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.1.1")
```

Comment:
this hasn't actually been released yet. I prefer to change it in the PR that actually does the release